action.yml

```diff
@@ -252,6 +255,7 @@ runs:
           && format('--python {0}', inputs.target-python-version)
           || ''
       }}
+      ${{ inputs.target }}
```

Comment:
I wonder if we should inject `--` before this.

Reply:
Ah, I wondered here if it was OK to just output the variable content. I was unsure what check or other to apply and follow what was already here with `${{ inputs.testing-type }}`.
What I though is that people using this action are already at a level where they can do whatever they want, so I didn't see the point in trying to protect.

What `--`  does ?

Reply:
> What `--` does ?

It's a common (and optional) way to explicitly separate the last part of the CLI args in POSIX programs. For example, if you have a filename called `--something.yml`, a CLI parser may think that it's a CLI option. But many CLI programs (and libs they are built with) allow doing `command-name --normal-arg --another-arg=value -- --something.yml` and then everything after `--` is treated as a list of positional arguments (typically files) that mustn't be treated as CLI options. This helps to disambiguate the invocation:
- https://unix.stackexchange.com/questions/11376/what-does-double-dash-mean
- https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html#tag_12_02

I haven't actually verified if `ansible-test` works with this but it would be reasonable to expect that it does.

Reply:
interesting, I used to use it to pass some parameters to subprogram, but didn't know about the POSIX rule.
Tks. 
I have no opinion :)

Reply:
I think we do need this explicit separator to avoid surprises.

Reply:
```suggestion
      --
      ${{ inputs.target }}
```

Reply:
Apparently, this is problematic: https://github.com/ansible-community/ansible-test-gh-action/issues/8#issuecomment-968072857.